src/atoms/listItem/component.tsx

```diff
@@ -65,6 +78,7 @@ export interface IListItemProps extends IComponentProps<IListItemTheme>, ISingle
   itemKey: string;
   isDisabled?: boolean;
   isSelected?: boolean;
+  dividerColor?: string;
```

Comment:
this should be in the theme. as should the width

Reply:
maybe we should just be using border-bottom for this?

Reply:
ah no but then the bottom one would have one too. in fact i think this maybe should be moved to the list component rather than here, otherwise it will always be drawn on the last one?

Reply:
and in fact that makes sense because its not part of the item, its between items

Reply:
not sure how, but also have a think about how the width and color can be given as part of the theme for a list (not as props)